File: conditionals/time.py
# river Stanley, how to get time of day
import time

#find first instance of time in programing
first_time= time.gmtime()
#print(first_time)

#current time in seconds 
current= time.time()
#print(current) #senonds since jan 1st 1970

#current date and time like we see it normally
now= time.ctime(current)
#print(now)

#just get a part of the time 
localtime = time.localtime(current)
day = localtime.tm_wday
hour = localtime.tm_hour #in military time  (0-23)
print(hour)

File: time.c
// River Stanley, getting the time


#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds; //time since jan 1 1970
    seconds = time(NULL);
    //printf("seconds jan 1st 1970 = %d", seconds);

    //current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    //printf("Current time and date is %s\n", asctime(timeinfo));

    //current hour 
    time_t now = time (NULL);

    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

    printf("%d\n", hour);


return 0;
}
